artifacts_test.py

```diff
@@ -341,10 +341,16 @@ def test_scylla_service(self):
                         "systemd-timesyncd is already installed, yet scylla installed ntp service on top of it"
                     assert not is_chrony_service_installed, \
                         "systemd-timesyncd is already installed, yet scylla installed chrony service on top of it"
-                else:
+                elif not self.params.get("unified_package"):
                     assert is_ntp_service_installed or is_chrony_service_installed, \
                         "systemd-timesyncd is not installed on the instance, yet Scylla did not install ntp or " \
                         "chrony services as a replacement"
+                else:
+                    # https://github.com/scylladb/scylla/issues/10608#issuecomment-1135770570
+                    # Scylla doesn't install any time sync service when it's an offline installation,
+                    # so if there's no time sync services active after the scylla installation, it's fine.
+                    # However, in such scenario a warning message should be printed during the scylla installation
+                    self.log.warning("No time sync service was installed. Passable since it's an offline installation.")
```

Comment:
@ShlomiBalalis , there is no need for such a warning.
We agreed with Avi that we should expect a warning from the installer output in this case.
No point having a warning in SCT log.

Reply:
I think it's fine to leave the warning in any case since our test should be independent and test the product

Reply:
@roydahan , this PR not following the suggested solution. i think it should be reverted.

Reply:
Right now we don't catch warnings from the setup, and I think our warning is good anyway.